Sources/PackageManagerDocs/Documentation.docc/Documentation.md

```diff
@@ -27,8 +27,8 @@ The Swift Package Manager leets you share your code as a package, depend on and
 <!-- - <doc:swift-docc-plugin> -->
 <!-- - <doc:swift-container-plugin> -->
 
-<!-- ### Reference -->
-<!-- link to PackageDescription API reference docc -->
+ ### Reference
+- <doc:PackageDescription> <!-- redirect page to PackageDescription API reference docc -->
```

Comment:
thought (non-blocking): instead of linking to the `PackageDescription.md`, can we link to the API documentation directly?  I'm just wondering value the contents of `PackageDescription.md` currently brings

Reply:
Yeah, that was the issue we ran into -- we're currently unable to directly link and re-direct to the other API docs, and this is currently the only feasible way to direct the user there. :')